solutions/usaco-916.mdx

```diff
@@ -66,6 +66,40 @@ public class Revegetation {
 ```
 
 </JavaSection>
+
+<PySection>
+
+```python
+import sys
+
+sys.stdin = open("revegetate.in", "r")
+sys.stdout = open("revegetate.out", "w")
+
+input = sys.stdin.readline
+
+n, m = map(int, input().split())
+
+a, b, g = [0] * 151, [0] * 151, [0] * 101
+
+for i in range(m):
+	a[i], b[i] = map(int, input().split())
+	if a[i] > b[i]:
+		a[i], b[i] = b[i], a[i]
+for i in range(1, n + 1):
```

Comment:
space stuff apart

also why are you doing line 78?

Reply:
making the program run a little faster? it should work either way.. i'll delete it

Reply:
noo, it's ok! ive just never seen that before